src/schema/v2/partner/__tests__/partner.test.js

```diff
@@ -1634,6 +1634,57 @@ describe("Partner type", () => {
   })
 
   describe("#alertsConnection", () => {
+    it("returns a single partner alert", async () => {
+      const response = {
+        body: {
+          id: "8754ff90-b020-425b-8dae-894ce5ad9d1f",
+          search_criteria_id: "3f980bbe-7b9b-4fa9-beb1-69d13e94fb0c",
+          partner_id: "5f80bfefe8d808000ea212c1",
+          search_criteria: {
+            id: "3f980bbe-7b9b-4fa9-beb1-69d13e94fb0c",
+            price_range: "1-2",
+          },
+        },
+      }
+
+      const query = gql`
+        {
+          partner(id: "catty-partner") {
+            alertsConnection(
```

Comment:
Ok pretty close, but still need to pass in a `first` argument
I think we want to be like: 'if id is present as an arg, dont worry about needing a first'